compareECLFiles.cmake

```diff
@@ -151,15 +151,16 @@ foreach(SIM flow flow_ebos flow_legacy)
                            TEST_ARGS tolerance_wells=1e-6 max_iter=20)
 endforeach()
 
-foreach(SIM flow flow_ebos flow_legacy)
+foreach(SIM flow flow_legacy)
   add_test_compareECLFiles(CASENAME spe9
                            FILENAME SPE9_CP_SHORT
                            SIMULATOR ${SIM}
                            ABS_TOL ${abs_tol}
                            REL_TOL ${rel_tol})
 endforeach()
 
-foreach(SIM flow flow_ebos)
+# TODO: Add legacy
```

Comment:
I don't think it's necessary to add a flow_legacy test for this, not even sure it would be succesful, @GitPaean do you agree?

Reply:
it runs with legacy, but i take your word. have removed the for and the todo comment.

Reply:
For this case, `flow_legacy` still works perfectly. It looks like there was no test for it, it is okay not to add a test now. I do not see future development for group control within `flow_legacy`.